javascript/src/detectors/javascript-avoid-nan-in-comparison/javascript-avoid-nan-in-comparison_compliant.js

```diff
@@ -0,0 +1,13 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+
+// {fact rule=javascript-avoid-nan-in-comparison@v1.0 defects=0}
+for (let i = 0; i < cars.length; i++) {
+    console.log(cars[i])
+    // Compliant : `isNaN()` function from Number will check for NaN value from array.
```

Comment:
could you pls change to `// Compliant: `

Reply:
we have strict compliant comment format validation in lib publish system. could you pls make sure your example follow the code example formate